HelloWorld/HelloWorld/ViewController.swift

```diff
@@ -23,7 +23,7 @@ class ViewController: UIViewController {
 
 
     @IBAction func buttonDidClick(_ sender: Any) {
-        helloLabel.text = "Super Hello, world. TEST!"
+        helloLabel.text = "Super Hello, world."
```

Comment:
Hi,
I think you need to follow our issue that is created by the PO.
Please revert it back.Thanks!